ui-guidelines/css/resources.css

```diff
@@ -0,0 +1,128 @@
+/*==== TYPOGRAPHY ====*/
+@import url(https://fonts.googleapis.com/css?family=Open+Sans);
```

Comment:
I'm not a big fan of adding internal `@import`s inside the CSS files. It may cause some unexpected issues if the file is broken or compromised.